### DISCUSSION TITLE
Running my app on natively Wayland in development

### DISCUSSION THREAD

**Discussion by amir-reza-bijandi** [alex8088/electron-vite#369] [General]
Is there any way to run my app natively under Wayland in development?
I know I should pass these arguments to electron executable, but I don't know how?

`--ozone-platform=wayland --enable-features=WaylandWindowDecorations`

**Comment by alex8088** [comment]
@amir-reza-bijandi, You can try updating to v2.0.0@beta, and append a `--` after the electron-vite CLI with the arguments to be passed.

```json
"scripts": {
  "dev": "electron-vite dev -- --ozone-platform=wayland --enable-features=WaylandWindowDecorations"
}
```

**Reply by mylinuxforwork** [reply]
I have the same issue. How can I update to v2.0.0@beta?

**Comment by amir-reza-bijandi** [comment]
@alex8088 
It worked! Tnx mate!
